lib/native/mqtt.spec.ts

```diff
@@ -3,42 +3,115 @@
  * SPDX-License-Identifier: Apache-2.0.
  */
 
-import { ClientBootstrap, Pkcs11Lib, TlsContextOptions } from '@awscrt/io';
-import { MqttClient, QoS } from '@awscrt/mqtt';
-import { AwsIotMqttConnectionConfigBuilder, WebsocketConfig } from '@awscrt/aws_iot';
-import { AwsCredentialsProvider } from '@awscrt/auth';
-import { Config, fetch_credentials } from '@test/credentials';
+import { ClientBootstrap, TlsContextOptions, ClientTlsContext, SocketOptions } from './io';
+import { MqttClient, MqttConnectionConfig, QoS } from './mqtt';
 import { v4 as uuid } from 'uuid';
+import {HttpProxyOptions, HttpProxyAuthenticationType, HttpProxyConnectionType} from "./http"
+import { AwsIotMqttConnectionConfigBuilder } from './aws_iot';
 
 jest.setTimeout(10000);
 
 const conditional_test = (condition: boolean) => condition ? it : it.skip;
 
-class Pkcs11Config {
-    public static LIB_PATH = process.env.AWS_TEST_PKCS11_LIB ?? ""
-    public static PIN = process.env.AWS_TEST_PKCS11_PIN ?? ""
-    public static TOKEN_LABEL = process.env.AWS_TEST_PKCS11_TOKEN_LABEL ?? ""
-    public static KEY_LABEL = process.env.AWS_TEST_PKCS11_KEY_LABEL ?? ""
-
-    public static is_valid() {
-        return Pkcs11Config.LIB_PATH !== "" &&
-            Pkcs11Config.PIN !== "" &&
-            Pkcs11Config.TOKEN_LABEL !== "" &&
-            Pkcs11Config.KEY_LABEL !== "";
+class AWS_IOT_ENV {
```

Comment:
I'm seeing AWS_IOT_ENV class being set up nearly if not identically in multiple files. Assuming this isn't something that can be set up once somewhere and shared?